packages/tools/src/utilities/calibrateImageSpacing.ts

```diff
@@ -9,25 +9,22 @@ const { calibratedPixelSpacingMetadataProvider } = utilities;
  * their reference imageIds. Finally, it triggers a re-render for invalidated annotations.
  * @param imageId - ImageId for the calibrated image
  * @param rowPixelSpacing - Spacing in row direction
- * @param columnPixelSpacing - Spacing in column direction
- * @param renderingEngine - Cornerstone RenderingEngine instance
+ * @param calibrationOrScale - either the calibration object or a scale value
  */
 export default function calibrateImageSpacing(
   imageId: string,
   renderingEngine: Types.IRenderingEngine,
-  rowPixelSpacing: number,
-  columnPixelSpacing: number
+  calibrationOrScale: Types.IImageCalibration | number
```

Comment:
The intent here is that it is a simple scale value, to agree (almost) with the old API, OR it is a full calibration object.  If it is still confusing, could you suggest a naming for it that explains that it is either a calibration object entirely or a scale value used to construct a calibration object?

Reply:
I mean you are changing the API from row/column pixelSpacing to a single one, so it is changing the API right now. I don't know if it really makes sense to change the API + make it backward compatible